websites/C/Coub/presence.ts

```diff
@@ -312,26 +295,27 @@ const pages: PageContext[] = [
 	],
 	presence = new Presence({
 		clientId: "818598086984728576",
-	});
-
-let currentLang: string, localizedStrings: { [key: string]: string };
-const IMAGES = {
-	PLAY: "playx1024",
-	PAUSE: "pausex1024",
-};
+	}),
+	presenceImageKeys = {
+		PLAY: "playx1024",
+		PAUSE: "pausex1024",
+	},
+	presenceStrings = {
+		browsing: "presence.activity.browsing",
+		watching: "presence.playback.playing",
+		watchVideo: "general.buttonWatchVideo",
+		viewProfile: "general.buttonViewProfile",
+	};
```

Comment:
This does not need to be a constant, it can just be passed onto the function

Reply:
Unresolved